server/app/services/mongo_pubsub.rb

```diff
@@ -93,7 +101,7 @@ def unsubscribe(subscription)
   def publish(channel, data)
     self.collection.insert_one(
       channel: channel,
-      data: data,
+      data: BSON::Binary.new(Marshal::dump(data)),
```

Comment:
oh, it actually kinda breaks things:
```
2.3.3 :008 > h = {foo: 'bar', message: ['foo', :foo, :bar], other: :other}
 => {:foo=>"bar", :message=>["foo", :foo, :bar], :other=>:other} 
2.3.3 :009 > packed = MessagePack.pack(h)
 => "\x83\xA3foo\xA3bar\xA7message\x93\xA3foo\xA3foo\xA3bar\xA5other\xA5other" 
2.3.3 :010 > unpacked = MessagePack.unpack(packed)
 => {"foo"=>"bar", "message"=>["foo", "foo", "bar"], "other"=>"other"} 
```
So it not only changes the hash keys but also values